src/qt/bitcoingui.cpp

```diff
@@ -1151,7 +1151,7 @@ void BitcoinGUI::setNumConnections(int count)
     default: icon = ":/icons/connect_4"; break;
     }
     labelConnectionsIcon->setPixmap(QIcon(icon).pixmap(STATUSBAR_ICONSIZE,STATUSBAR_ICONSIZE));
-    labelConnectionsIcon->setToolTip(tr("%n active connection(s) to Gridcoin network", "", count));
+    labelConnectionsIcon->setToolTip(tr("%1 active connection(s) to Gridcoin network").arg(count));
```

Comment:
What do you mean by breaks the existing translations? I tested it with german, english and spanish and it worked well.